### DISCUSSION TITLE
Change app banner

### DISCUSSION THREAD

**Discussion by lykaodan** [Budibase/budibase#4012] [Help]
Hello! I just want to ask if I can possibly change this app banner? 

![image](https://user-images.githubusercontent.com/32080504/149259980-e56796b9-ee04-4b88-a763-5b1d66c793d1.png)

**Comment by joebudi** [comment]
We've updated the homepage design in the new release. You'll be able to represent each app with an icon. This will be released in the next few days.

**Comment by lykaodan** [comment]
I'm looking forward to it. Thank you!
